sopel/modules/url.py

```diff
@@ -39,7 +47,7 @@
 # the title. We don't want it too high, or a link to a big file/stream will
 # just keep downloading until there's no more memory. 640k ought to be enough
 # for anybody.
```

Comment:
```suggestion
# just keep downloading until there's no more memory. 640k ought to be enough
# for anybody, but the modern web begs to differ.
```

It's not 640k any more, so the original comment is no longer accurate 😭

Did you add `* 2` instead of changing the number below so 640k would still "be there"? 😹

Reply:
You caught me red-handed lol